### DISCUSSION TITLE
Is macOS-13 hosted runner still in beta?

### DISCUSSION THREAD

**Discussion by mariusfanu-metrobank** [actions/virtual-environments#8949] [General questions] [answered]
Hi, 

This might be a silly question, but is macOS 13 agent still in beta or it's fully supported?

To give a bit of context, we use Azure Pipelines and on the [Microsoft website](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software) it's not tagged as beta
<img width="833" alt="Screenshot 2023-12-05 at 11 16 14" src="https://github.com/actions/runner-images/assets/129301957/ee366a29-d02f-4eb6-9b6f-9276f8477c3f">

On the other side on [GitHub](https://github.com/actions/runner-images#available-images) it is still tagged as being beta, as seen below:
<img width="871" alt="Screenshot 2023-12-05 at 11 16 26" src="https://github.com/actions/runner-images/assets/129301957/b1766958-ebfa-4dd8-9251-dadd65c816a8">

Thanks

**Comment by Cheesebaron** [comment, marked as answer]
According to: https://developercommunity.visualstudio.com/t/macos-13-Pool-provider-seems-slow/10522135#T-N10522800 it is still in beta

**Comment by mariusfanu-metrobank** [comment]
Thanks, that is really unfortunate. I hope they'll take it out of beta soon and give us macOS 14 as well...
